### PR TITLE
Issue #3002: [UX] Improvements to pathauto bulk-update page

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/backdrop/backdrop-issues/issues/3002

### CODE REVIEW

File: core/modules/path/path.admin.inc [hunk 1]
```diff
@@ -628,7 +628,7 @@ function path_bulk_update_form($form, &$form_state) {
     if (!empty($path_info['batch update callback'])) {
       $form['update'][$type]['base'] = array(
         '#type' => 'checkbox',
-        '#title' => t('!label URL aliases', array('!label' => $path_info['label'])),
+        '#title' => $path_info['alias bulk actions description'],
```
Comment (opi) [inline core/modules/path/path.admin.inc:631]: `alias bulk actions description` is a new required key for https://api.backdropcms.org/api/backdrop/core%21modules%21path%21path.api.php/function/hook_path_info/1 , so maybe this is an API break.
At least hook_path_info documentation must be updated in https://github.com/backdrop/backdrop/blob/1.x/core/modules/path/path.api.php#L104

File: core/modules/path/path.admin.inc [hunk 2]
```diff
@@ -640,11 +640,11 @@ function path_bulk_update_form($form, &$form_state) {
       // a row for each of these bundle types. For the Node entity the pattern
       // items would be keyed by be node types; for taxonomy, these would be
       // vocabularies.
-      if (isset($path_info['pattern items'])) {
-        foreach ($path_info['pattern items'] as $item => $item_label) {
+      if (isset($path_info['alias bulk actions items'])) {
```
Comment (opi) [inline core/modules/path/path.admin.inc:643]: same here

**Review by opi — CHANGES_REQUESTED** [review]

File: core/modules/path/path.api.php [hunk 3]
```diff
@@ -116,7 +116,11 @@ function hook_path_delete($path) {
  *   - label: Translated label for the settings group.
  *   - pattern description: The translated label for the default URL alias
  *       pattern (e.g. t('Default path pattern (applies to all content types
- *       with blank patterns)')
+ *       with blank patterns)').
+ *   - alias bulk actions description: The translated label for the checkboxes
```
Comment (opi) [inline core/modules/path/path.api.php:120]: Nice. You may add the `alias bulk actions items` key too

File: core/modules/path/path.path.inc [hunk 4]
```diff
@@ -146,7 +149,8 @@ function taxonomy_path_info() {
   if (count($vocabularies)) {
     $info['taxonomy_term']['pattern items'] = array();
     foreach ($vocabularies as $vocabulary_name => $vocabulary) {
-      $info['taxonomy_term']['pattern items'][$vocabulary_name] = t('URL alias pattern for all %vocab-name URLs', array('%vocab-name' => $vocabulary->name));
+    $info['taxonomy_term']['pattern items'][$vocabulary_name] = t('URL alias pattern for all %vocabulary_label URLs', array('%vocabulary_label' => $vocabulary->name));
+    $info['taxonomy_term']['alias bulk actions items'][$vocabulary_name] = t('URL alias for <strong>%vocabulary_label</strong> <small>(Machine name: <code>!machine_name</code>)</small>', array('%vocabulary_label' => $vocabulary->name, '!machine_name' => $vocabulary_name));
```
Comment (quicksketch) [inline core/modules/path/path.path.inc:153]: Needs another two spaces indentation.

**Comment by opi** [comment]
rerolled https://github.com/backdrop/backdrop/pull/2310

**Comment by opi** [comment]
can we close this PR ?

**Comment by klonos** [comment]
...yep, sorry for the late response @opi 

Closing in favour of https://github.com/backdrop/backdrop/pull/2310

**Pull request closed without merging**